templates/tfengine/components/audit/main.tf

```diff
@@ -23,14 +23,12 @@ terraform {
 }
 
 {{- $parent_field := "org_id"}}
-{{- $parent_var := "var.org_id"}}
```

Comment:
The reason I preferred to use vars here was to reduce duplication a bit in the generated configs. Should we keep the vars?

Reply:
Could you elaborate a little on how it reduce duplication in the generated configs?

And also in a lot of places we are directly use values in `main.tf`, not through the `terraform.tfvars`.

Reply:
Yeah in general I think we are now preferring to hard code values within main.tf. I have still preferred to use variables for the primary parent id, for example all resources still use var.project_id within projects: https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/examples/tfengine/generated/team/live/example-prod-data/resources/main.tf#L32.

Since parent ids are static throughout the deployment it's relatively safe to make them vars so that you don't see the number repeated so many times throughout the deployment. However if you want to just have one way of doing things and just putting the id directly in main.tf that's fine with me too.

Reply:
Discussed offline. Use vars.